.travis.yml

```diff
@@ -1,6 +1,7 @@
 branches:
   only:
   - master
+  - add_system_vars_to_driver_recording
```

Comment:
This should not remain in our real .travis.yml

Reply:
fixed and pushed up again